File: CodeForces/Photos.go
// Problem: https://codeforces.com/problemset/problem/707/A

package main

import "fmt"

func main(){
	var rows int
	var cols int
	fmt.Scanf("%d %d", &rows, &cols)

	var ch byte
	for i:=0; i<rows+cols; i++ {
		fmt.Scanf("%s", &ch)
		fmt.Println("Scanned: ", ch)
		if(ch == 'C' || ch == 'M' || ch == 'Y'){
			fmt.Println("#Color")
			return;
		}
	}
	fmt.Println("#Black&White")
}

File: CodeForces/Photos.java
// https://codeforces.com/problemset/problem/707/A
import java.util.Scanner;
public class Photos{
    public static void main(String[] args){
        Scanner sObj = new Scanner(System.in);
        int row = sObj.nextInt();
        int col = sObj.nextInt();
        sObj.nextLine();
        for(int i=0; i<row; i++){
            for(char ch: sObj.nextLine().toCharArray()){
                if(ch == 'C' || ch == 'M' || ch == 'Y'){
                    System.out.println("#Color");
                    return;
                }
            }
        }
        System.out.println("#Black&White");
    }
}